src/player.py

```diff
@@ -1,2 +1,32 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+class Player:
+    def __init__(self, name, current_room):
+        self.name = name
+        self.current_room= current_room
+        self.items = []
+
+    def print_info(self):
+        makeString = ''.join(str(x) for x in self.items)
+        print(f"{self.name}, Your items:{makeString} {self.current_room}")
+        
+
```

Comment:
This works, but I don't feel like it's quite up to the spec.  Looks like it's all or nothing.  How might you be able to move individual items around?